packages/editor/src/utils/media-upload/validateFileSize.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { select } from '@wordpress/data';
+import { validateFileSize as _validateFileSize } from '@wordpress/media-utils';
+
+/**
+ * Internal dependencies
+ */
+import { store as editorStore } from '../../store';
+
+/**
+ * Verifies whether the file is within the file upload size limits for the site.
+ *
+ * @param file File object.
+ */
+export function validateFileSize( file: File ) {
```

Comment:
Can we make this util to get `maxUploadFileSize` as a param to avoid the store subscription? 

Same question for `validateMimeType` function below.